src/components/AirDateBadge/index.tsx

```diff
@@ -38,7 +39,7 @@ const AirDateBadge = ({ airDate }: AirDateBadgeProps) => {
           year: 'numeric',
           month: 'long',
           day: 'numeric',
-          timeZone: 'UTC',
+          timeZone,
```

Comment:
@gauthier-th No need to offset even. The relative time is already calculated correctly. Just have to pass in the proper timezone